server/text_generation_server/models/causal_lm.py

```diff
@@ -122,7 +122,7 @@ def from_pb(
         position_ids.masked_fill_(tokenized_inputs["attention_mask"] == 0, 1)
         all_input_ids = tokenized_inputs["input_ids"].T.split(1, dim=1)
 
-        max_tokens = len(inputs) * max_input_length + max_decode_tokens
+        max_tokens = len(inputs) * (max_input_length + max_decode_tokens)
```

Comment:
HI, Dehaene. max_decode_tokens is cumulated in the for loop before. Why times len(inputs) here?